modules/sara_r4/cellular_r4_api.c

```diff
@@ -2736,6 +2739,29 @@ CellularError_t Cellular_SetPsmSettings( CellularHandle_t cellularHandle,
 
 /*-----------------------------------------------------------*/
 
+static CellularError_t _Cellular_isSockOptSupport( CellularSocketOptionLevel_t optionLevel,
+                                                   CellularSocketOption_t option )
+{
+    CellularError_t err = CELLULAR_UNSUPPORTED;
+
+    if( ( optionLevel == CELLULAR_SOCKET_OPTION_LEVEL_TRANSPORT ) &&
+        ( ( option == CELLULAR_SOCKET_OPTION_SEND_TIMEOUT ) ||
+          ( option == CELLULAR_SOCKET_OPTION_RECV_TIMEOUT ) ||
+          ( option == CELLULAR_SOCKET_OPTION_PDN_CONTEXT_ID ) ) )
+    {
+        err = CELLULAR_SUCCESS;
+    }
+    else
+    {
+        LogError( ( "Cellular_SocketSetSockOpt: Option [Level:option=%d:%d] not supported in SARA R4",
```

Comment:
In a function which tells whether or not a socket option is supported, this is not error. This log should be changed to debug.